src/asset_builder.rb

```diff
@@ -42,17 +42,7 @@ def edit_asset(name)
     private
 
     def source_file_details(layout_or_type)
```

Comment:
Now that the code has been moved to `Records::Layout`. We no longer need this method.

Reply:
Removed the method, it now calls `Records::Layout.type_or_layout` directly from `push_asset`